sklearn/decomposition/_truncated_svd.py

```diff
@@ -59,6 +61,18 @@ class TruncatedSVD(_ClassNamePrefixFeaturesOutMixin, TransformerMixin, BaseEstim
         :func:`~sklearn.utils.extmath.randomized_svd` to handle sparse
         matrices that may have large slowly decaying spectrum.
 
+    n_oversamples : int, default=10
+        Number of oversamples for randomized SVD solver. Not used by ARPACK.
+        See :func:`~sklearn.utils.extmath.randomized_svd` for a complete
+        description.
+
+        .. versionadded:: 1.1
```

Comment:
This newline is to be consistent with the style of the rest of the codebase that uses `versonadded`:

```suggestion
        .. versionadded:: 1.1
        
```